javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,49 @@
+var charNodes = document.querySelectorAll('span');
+var currentIndex = 0;
+document.addEventListener('keydown', typingTest);
+
+function typingTest(event) {
+  if (currentIndex === charNodes.length) {
+    alert('DONE! Restarting Test....');
+    window.location.reload();
+  }
+  if (event.keyCode !== 8) {
+    checkCharacterIsValid(event.key, currentIndex);
+  }
+  if (event.key === charNodes[currentIndex].innerHTML || event.keyCode === 8) {
```

Comment:
`event.keyCode === 8` 

What is this for?

Reply:
I used this for backspace key.  Am i not allowed to use that?